contracts/contracts/rem.system/src/voting.cpp

```diff
@@ -40,6 +40,8 @@ namespace eosiosystem {
 
       user_resources_table totals_tbl( _self, producer.value );
       const auto& tot = totals_tbl.get(producer.value, "producer must have resources");
+      const auto &voter = _voters.get(producer.value, "user has no resources");
```

Comment:
In does_satisfy_stake_requirement( producer ) lookup of voters table is executed.
Here you added _voters.get(producer.value, "user has no resources");

Consider removing does_satisfy_stake_requirement
Something like this:
const auto &voter = _voters.get(producer.value, "user has no resources");
check( voter->staked >= producer_stake_threshold, stake_err );